src/main/java/bio/terra/service/filedata/DrsService.java

```diff
@@ -184,10 +242,13 @@ private DRSObject drsObjectFromFSFile(
 
     DRSAccessURL gsAccessURL = new DRSAccessURL().url(fsFile.getGspath());
 
+    GoogleRegion region = bucketResource.getRegion();
+    String accessId = "gcp-" + region.getValue();
```

Comment:
this is fine for now but eventually we should pull this from the snapshot's cloud platform